.github/workflows/monthly.yml

```diff
@@ -0,0 +1,41 @@
+name: monthly
+
+on:
+  push
+    branches:
+      - master
+
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  run:
+    name: Refresh month of man pages
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
```

Comment:
You don't have a `matrix` here, use the latest ruby directly: `ruby-version: 2.7.1`.

Reply:
Eeks! My bad.
Okay, I  can use 2.7.1, of course. Thanks! :smile: